CHANGELOG.md

```diff
@@ -25,6 +25,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Clicking on URLs on Windows incorrectly opens File Explorer
 - Incorrect underline cursor thickness on wide cell
 - Viewport moving around when resizing while scrolled into history
+- macOS keybinding (cmd+alt+H) hiding all windows except Alacritty
```

Comment:
This is an addition, not a fix. Also I think phrasing it `all windows other than Alacritty` is a little easier to understand, since it signifies more properly that we're talking about this specific Alacritty instance and not Alacritty in general.